test/environments/kitchen.rb

```diff
@@ -28,4 +28,16 @@
   'raid_unmount-raid_vol_array/ubuntu1804' => '',
   'raid_unmount-raid_vol_array/ubuntu2004' => '',
   'raid_unmount-raid_vol_array/ubuntu2204' => '',
+  'lustre_mount-fsx_fs_id_array' => ["fs-0ab11b3ade43091fe"],
+  'lustre_mount-fsx_fs_type_array' => ["LUSTRE"],
+  'lustre_mount-fsx_shared_dir_array' => ["shared_dir"],
+  'lustre_mount-fsx_dns_name_array' => ["fs-0ab11b3ade43091fe.fsx.us-west-2.amazonaws.com"],
+  'lustre_mount-fsx_mount_name_array' => ["qz5b7bev"],
+  'lustre_mount-fsx_volume_junction_path_array' => [""],
+  'lustre_unmount-fsx_fs_id_array' => '',
+  'lustre_unmount-fsx_fs_type_array' => '',
```

Comment:
We don't need to define all these variables in the kitchen.rb most of them are not variable that are coming from FSx resource.

We can remove for sure `fsx_fs_type_array`, `fsx_shared_dir_array`, `fsx_dns_name_array` (the prefix is the id_array), etc

I mean we can have most of them hardcoded in the hooks and in the test and have here just the `fsx_fs_id_array` variable or the other info coming from the hooks/AWS.

Reply:
ah ok, makes sense.  I will make this change